### PR TITLE
done login UI

### PR DESCRIPTION
**Pull request by Harshtroo**

### CODE REVIEW

File: tms/clickup/urls.py [hunk 1]
```diff
@@ -3,7 +3,8 @@
 
 urlpatterns = [
     path("", views.get_home_page, name="home"),
-    path("get_register/",views.get_register_page,name="get_register"),
+    path("get_register/", views.get_register_page, name="get_register"),
```
Comment (yatishTrootech) [inline tms/clickup/urls.py:6]: register

File: tms/clickup/urls.py [hunk 2]
```diff
@@ -3,7 +3,8 @@
...
+    path("login_page/", views.get_login_page, name="login_page"),
```
Comment (yatishTrootech) [inline tms/clickup/urls.py:7]: login

File: tms/clickup/views.py [hunk 3]
```diff
@@ -19,6 +19,10 @@ def get_register_page(request):
     return render(request, "register.html", {"group": group})
 
 
+def get_login_page(request):
```
Comment (yatishTrootech) [inline tms/clickup/views.py:24]: login

File: tms/clickup/views.py [hunk 4]
```diff
@@ -27,7 +31,8 @@ def create(self, request, *args, **kwargs):
         serializer = self.get_serializer(data=request.data)
         serializer.is_valid(raise_exception=True)
         self.perform_create(serializer)
-        return JsonResponse({"data": serializer.data, "status": "success"}, status=status.HTTP_201_CREATED)
+        return JsonResponse({"data": serializer.data, "status": "success", "success_message": "Register successfully"},
```
Comment (yatishTrootech) [inline tms/clickup/views.py]: return Response

**Review by yatishTrootech — CHANGES_REQUESTED** [review]

File: tms/clickup/views.py [hunk 5]
```diff
@@ -36,9 +41,10 @@ def post(self, request):
         serializer.is_valid(raise_exception=True)
         user = serializer.validated_data
         token, created = Token.objects.get_or_create(user=user)
-        return JsonResponse({'token': token.key,
+        return JsonResponse({"token": token.key,
+                             "status": "success",
                              "user": user.username,
-                             "message": "Successfully logged in."}, status=status.HTTP_200_OK)
+                             "success_message": "Successfully logged in."}, status=status.HTTP_200_OK)
```
Comment (Dhruvi-Makwana) [inline tms/clickup/views.py]: add message in constant.

**Review by yatishTrootech — APPROVED** [review]

**Pull request merged**